pcsx2/gui/AppMain.cpp

```diff
@@ -547,18 +570,35 @@ void Pcsx2App::LogicalVsync()
 	FpsManager.DoFrame();
 	
 	if (EmuConfig.Gamefixes.FMVinSoftwareHack) {
-		if (EnableFMV == 1) {
+		if (EnableFMV) {
 			Console.Warning("FMV on");
 			DoFmvSwitch();
-			EnableFMV = 0;
+			EnableFMV = false;
```

Comment:
This will make your toggling only work if you did not enable the switch to sw hack. I would suggest to merge the enabling and disabling of both hacks


Maybe something like that:
```
if (EmuConfig.Gamefixes.FMVinSoftwareHack || g_Conf->GSWindow.IsToggleAspectRatioSwitch) {
    if (EnableFMV) {
        Console.Warning("FMV on");
        if (EmuConfig.Gamefixes.FMVinSoftwareHack) 
            DoFmvSwitch();
        if (g_Conf->GSWindow.IsToggleAspectRatioSwitch)
            Do43Switch();
        EnableFMV = false;
    }
}
```

and the same for the disabling.

If it was intended to keep the aspect ratio on sw renderer ignore me.